website/content/docs/secrets/gcp.mdx

```diff
@@ -40,22 +40,77 @@ management tool.
 
 1.  Enable the Google Cloud secrets engine:
 
-    ```shell-session
+<Tabs>
+
+<Tab heading="CLI">
+
+   ```shell-session
     $ vault secrets enable gcp
     Success! Enabled the gcp secrets engine at: gcp/
     ```
 
     By default, the secrets engine will mount at the name of the engine. To
     enable the secrets engine at a different path, use the `-path` argument.
+</Tab>
+
+<Tab heading="GUI">
+
+![Partial screenshot of the Vault GUI showing the "Identity token key" search select input](/img/gui/gcp/identityTokenKey.png)
+
+You must have `list` permission on the `identity/oidc/key` endpoint to view
+existing workload identity federation (WIF) identity token keys during setup.
+
+1. Open the web UI for your Vault instance.
+1. Login under the target namespace or choose the target namespace from the
+   selector at the bottom of the left-hand menu and re-authenticate.
+1. Select **Secrets Engines** from the left-hand menu.
+1. Click **Enable new engine +** on the **Secrets Engines** page.
+1. Select **Google Cloud**.
+1. Click **Next**.
+1. Set the mount path for the GCP plugin. For example, `gcp`.
+1. If you use WIF, add the identity token key:
+    - Click **Method Options**.
+    - Click **Identity Token Key**. 
+    - Enter your new key name or select one from the token key list.
+1. Click **Enable engine**.
+1. Click **Save** to enable the plugin.
+
+</Tab>
+
+</Tabs>
 
 1.  Configure the secrets engine with account credentials, or leave blank or unwritten
     to use Application Default Credentials.
 
-    ```shell-session
+<Tabs>
+
+<Tab heading="CLI">
+
+   ```shell-session
```

Comment:
Remove indents. For example:
````
```shell-session
$ vault write gcp/config credentials=@my-credentials.json
Success! Data written to: gcp/config
```
````